Watchman.Discord/Areas/Responses/Controllers/ResponsesController.cs

```diff
@@ -106,5 +110,25 @@ public async Task RemoveResponse(DiscordRequest request, Contexts contexts)
             await _responsesService.RemoveResponse(onEvent, contexts.Server.Id);
             await messageService.SendResponse(x => x.ResponseHasBeenRemoved(contexts, onEvent), contexts);
         }
+
+        [AdminCommand]
+        [DiscordCommand("responses")]
+        public async Task Responses(DiscordRequest request, Contexts contexts)
+        {
+            var argument = request.Arguments?.FirstOrDefault()?.Name;
+            string commandArgument = null;
+
+            if (argument == null)
+            {
+                commandArgument = "all";
+            }
+            else if (argument != "all" && argument != "default" && argument != "custom")
+            {
+                throw new InvalidArgumentsException("-all -default -custom");
+            }
+            commandArgument = commandArgument ?? argument;
```

Comment:
```csharp
var commandArgument = request.Arguments?.FirstOrDefault()?.Name.ToLowerInveriant();
if (!possibleArguments.Contains(commandArgument))
{
    commandArgument = "all";
}
```

wtedy nawet jak ktoś poda zły argument, to wpadnie domyślny i wyjątek nie będzie potrzebny

Reply:
possibleArguments chyba zrobię jako lokalną tablicę, bo nie wiem czy mamy jakoś bazę czy coś. Ten InvalidArgumentsException to nie będę go tutaj wyrzucał, ale niech zostanie w projekcie, może kiedyś się przyda w innym poleceniu.